history-server.md

```diff
@@ -0,0 +1,71 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
+* [kubens](https://github.com/ahmetb/kubectx)
+
+### Installing Spark Operator with History Server Enabled
+
+```
+kubens spark-operator
+kubectl kudo install ./kudo-operator -p operatorImageName=mesosphere/kudo-spark-operator -p operatorVersion=latest -p enableHistoryServer=true
+```
+
+This will setup Spark History Server Pod and Service with Spark Event Log directory pointing to a location `/opt/spark/work-dir` (which you can change by passing another location to a parameter `historyServerFsLogDirectory`). This location is local to a Pod of Spark History Server. If we wnat to make it persistent and shared between different Pods, we have to create a PVC (Persistent Volume Claim) and pass it while starting the Spark History Server.
```

Comment:
```suggestion
This will deploy a Pod and Service for the `Spark History Server` with the `Spark Event Log` directory configured via the `historyServerFsLogDirectory` parameter. By default, the directory is set to  `/opt/spark/work-dir`. This location is local to the `Spark History Server` pod. If users want to persist and share the data between different pods, users must create a Persistent Volume Claim (PVC) and specify that claim in the Spark History Server manifest.
```